util/src/test/java/io/kubernetes/client/util/ClientBuilderTest.java

```diff
@@ -76,6 +77,14 @@ public void testDefaultClientReadsKubeConfig() throws Exception {
     assertEquals("http://kubeconfig.dir.com", client.getBasePath());
   }
 
+  @Test
+  public void testDefaultClientReadsKubeConfigMultiple() throws Exception {
+    final String kubeConfigEnv = KUBECONFIG_FILE_PATH + File.pathSeparator + KUBECONFIG_FILE_PATH;
```

Comment:
Let's make the second file something different like `/doesn't/exist` or somesuch to validate the "chooses first" is correct.

Reply:
fixed